src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -269,7 +269,8 @@ function handleUnderworldMarket(cardList, playerColor)
       card.zone = "SetAside3"
     elseif (card.metadata.traits ~= nil and string.find(card.metadata.traits, "Illicit", 1, true)
         and card.metadata.bonded_to == nil
```

Comment:
Might want to fix this while you're at it.  We don't populate bonded_to any more.  I think this and the permanent check can be combined into a zone check to only pull cards from the Deck.